graylog2-server/src/main/java/org/graylog2/rest/resources/streams/DefaultStreamDeletionGuard.java

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright (C) 2020 Graylog, Inc.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the Server Side Public License, version 1,
+ * as published by MongoDB, Inc.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * Server Side Public License for more details.
+ *
+ * You should have received a copy of the Server Side Public License
+ * along with this program. If not, see
+ * <http://www.mongodb.com/licensing/server-side-public-license>.
+ */
+package org.graylog2.rest.resources.streams;
+
+import jakarta.inject.Singleton;
+
+@Singleton
+public class DefaultStreamDeletionGuard implements StreamDeletionGuard {
+
+    @Override
+    public void checkGuard(String streamId) {
+        // Do nothing in graylog open
```

Comment:
Why do you add a default guard if it doesn't do anything?

Reply:
you are right, just declaring the Multibinder is enough, fixing that